client/react/frontpage/components/SpaceJam.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './SpaceJam.scss';
+
+class SpaceJam extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  changeBackground(newB) {
```

Comment:
The background change is still a little glitchy as sometimes when I click on an alien after repeatedly changing it all I get is a white background. 

I'm not sure what the exact cause might be but we can explore a later solution as this develops.